lambdatimes/src/components/Content/Content.js

```diff
@@ -17,38 +17,25 @@ export default class Content extends Component {
   }
 
   componentDidMount() {
-    // Once the component has mounted, get the data and reflect that data on the state.
+    this.setState(() => ({tabs: tabData, cards: cardData}));
```

Comment:
You don't have to scope `setState` or use an anonymous function for it. You can pass `state` as a parameter and then you could use it there but the way you have it `this.setState(({tabs: tabData, cards: cardData}));` Would be perfect